util/zib2fsh/zib2fsh.py

```diff
@@ -250,27 +299,40 @@ def asCore(self):
                 else:
                     el.comment = comment
 
-            if el.min == "0":
+            # For slices that are already defined in the base profile, we must clear their
+            # cardinality to prevent them from being defined again in the core profile.
+            if el.slice_name and not el.slicing_path:

```

Comment:
I think this is incorrect, as it removes the cardinality constraints from slices and extensions, which should be added here. Where is this resulting in unwanted output?